pulpcore/content/handler.py

```diff
@@ -135,11 +136,11 @@ def _permit(request, distribution):
                     'g': guard.name,
                     'r': str(pe)
                 })
-            raise
+            raise HTTPForbidden(reason=str(pe))
```

Comment:
Is there any chance this will be used outside of the Content app?

Reply:
Really can't think of a use case.  Can you?

Reply:
I can't think of a reason, but if a plugin writer decided to offer content validation through the rest API for whatever reason, they would be using Django there and catching an aiohttp exception would be strange, but a PermissionError wouldn't. Transforming it in the content app handler, I think makes the ContentGuard feature a bit more portable.

Reply:
Actually what if we wanted a feature that validated X509 certs as they are created that they are in turn valid w/ some uber cert. This would be a possible future feature that could make sense. That validation would happen in Django.

Reply:
In both these cases, a plugin writer using the `Handler._permit()` separately for this _seems_  inappropriate (or at least not a good fit) because:
- It's a protected method (which could be changed)
- There is no guarantee that the associated content guard is X.509.
- I would think that anything leveraging the content guard would be using (or subclassing) the `Handler` anyway.

Reply:
That's a good point, why is `_permit()` private? Is there a public version of it?

Reply:
I was confusing which object permit() was on. After looking at it again I can see the implementation here is all correct.